src/server/getters/get-dispute-info.ts

```diff
@@ -138,6 +146,7 @@ function reshapeStakeRowToUIStakeInfo(stakeRows: DisputesResult): UIStakeInfo|nu
   const disputeRound = totalCompletedStakeOnAllPayouts.equals(0) ? null : stakeRows.disputeRound[0] == null ? 0 : stakeRows.disputeRound[0].disputeRound;
   return {
     marketId: marketRow.marketId,
+    stakeCompletedTotal: totalCompletedStakeOnAllPayouts.toFixed(),
```

Comment:
This is the total stake for the last completed dispute bond? Or is it the stake needed to complete the current active disptue?

Reply:
It is the sum of all currently completed stake, including initial, so you will need to double it. I could double add a double'd version if you like.

Reply:
Could you add a stakeCurrentTotal that would be the current dispute bond total? Does that make sense? 
stakeCurrentTotal would be double the stakeCompletedTotal. So if the market is in awaiting fee window after designated reporter then stakeCompletedTotal would be the DR stake and stakeCurrentTotal would be double stakeCompletedTotal, correct?